templates/search/blocks/search_results_tag_frequency.html

```diff
@@ -5,41 +5,46 @@
         <p class="tag-frequency__title">Filter by:</p>
         <ul class="tag-frequency__tags-list">
             <li>
-                <a href="?{% query_string_include 'chart_data_type' 'LOC' %}" class="ohos-tag ohos-tag--location">
+<!--                <a href="?{% query_string_include 'chart_data_type' 'LOC' %}" class="ohos-tag ohos-tag&#45;&#45;location">-->
+                    <a href="/search/catalogue/?group=community&vis_view=tag&chart_data_type=LOC" class="ohos-tag ohos-tag--location">
                     <span class="ohos-tag__inner">Location</span>
                 </a>
             </li>
 
             <li>
-                <a href="?{% query_string_include 'chart_data_type' 'PER' %}" class="ohos-tag ohos-tag--person">
+<!--                <a href="?{% query_string_include 'chart_data_type' 'PER' %}" class="ohos-tag ohos-tag&#45;&#45;person">-->
+                    <a href="/search/catalogue/?group=community&vis_view=tag&chart_data_type=PER" class="ohos-tag ohos-tag--person">
                     <span class="ohos-tag__inner">Person</span>
                 </a>
             </li>
 
             <li>
-                <a href="?{% query_string_include 'chart_data_type' 'ORG' %}" class="ohos-tag ohos-tag--organisation">
+<!--                <a href="?{% query_string_include 'chart_data_type' 'ORG' %}" class="ohos-tag ohos-tag&#45;&#45;organisation">-->
+                    <a href="/search/catalogue/?group=community&vis_view=tag&chart_data_type=ORG" class="ohos-tag ohos-tag--organisation">
                     <span class="ohos-tag__inner">Organisation</span>
                 </a>
             </li>
 
             <li>
-                <a href="?{% query_string_include 'chart_data_type' 'MISC' %}" class="ohos-tag ohos-tag--misc">
+<!--                <a href="?{% query_string_include 'chart_data_type' 'MISC' %}" class="ohos-tag ohos-tag&#45;&#45;misc">-->
+                    <a href="/search/catalogue/?group=community&vis_view=tag&chart_data_type=MISC" class="ohos-tag ohos-tag--misc">
```

Comment:
same as above on commented and link.

Reply:
The commented code has now been removed.